judge/models/contest.py

```diff
@@ -346,6 +346,7 @@ class ContestParticipation(models.Model):
     cumtime = models.PositiveIntegerField(verbose_name=_('cumulative time'), default=0)
     is_disqualified = models.BooleanField(verbose_name=_('is disqualified'), default=False,
                                           help_text=_('Whether this participation is disqualified.'))
+    tiebreaker = models.FloatField(verbose_name=_('tie-breaking field'), default=0.0)
```

Comment:
`tiebreaker` should probably be included [here](https://github.com/DMOJ/online-judge/blob/master/judge/views/contests.py#L583) as well to sort the actual rankings. (Unless I'm misunderstanding the ICPC format)